src/Symfony/Component/AssetMapper/Command/AuditCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Symfony\Component\AssetMapper\Command;
+
+use Symfony\Component\AssetMapper\ImportMap\ImportMapAuditor;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(name: 'importmap:audit', description: 'Checks for security vulnerability advisories for dependencies.')]
+class AuditCommand extends Command
+{
+    private const SEVERITY_COLORS = [
+        'info' => 'default',
+        'low' => 'default',
+        'moderate' => 'yellow',
+        'high' => 'red',
+        'critical' => 'red',
+    ];
+
+    public function __construct(
+        private readonly ImportMapAuditor $importMapAuditor,
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $audit = $this->importMapAuditor->audit();
+
+        if ([] === $audit) {
+            $io->info('No packages to audit.');
+
+            return self::SUCCESS;
+        }
+
+        $table = $io->createTable();
+        $table->setHeaders([
+            'Severity',
+            'Title',
+            'Package',
+            'Version',
+            'Patched in',
+            'More info',
+        ]);
+
+        foreach ($audit['advisories'] as $advisory) {
+            $table->addRow([
+                sprintf('<fg=%s>%s</>', self::SEVERITY_COLORS[$advisory['severity']] ?? 'default', ucfirst($advisory['severity'])),
+                $advisory['title'],
+                $advisory['module_name'],
+                $audit['packages'][$advisory['module_name']] ?? 'n/a',
+                $advisory['patched_versions'],
+                $advisory['url'],
+            ]);
+        }
+        $table->render();
```

Comment:
what if there is no advisories ? Rendering table headers without any rows looks weird to me.

Reply:
Good catch. Done.